src/olympia/blocklist/cron.py

```diff
@@ -0,0 +1,25 @@
+import json
+import tempfile
+
+import olympia.core.logger
+from olympia.lib.kinto import KintoServer
+
+from .mlbf import generate_mlbf, get_mlbf_key_format
+from .utils import KINTO_BUCKET, KINTO_COLLECTION_MLBF
+
+log = olympia.core.logger.getLogger('z.cron')
+
+
+def upload_mlbf_to_kinto():
```

Comment:
This should probably be behind a waffle switch for now.

Reply:
https://github.com/mozilla/addons-server/pull/13659/commits/d34fd01b586cafd8a53855d9b5dc1c1e9056074a